Orange/classification/elliptic_envelope.py

```diff
@@ -0,0 +1,39 @@
+import sklearn.covariance as skl_covariance
+
+from Orange.classification import SklLearner
+
+__all__ = ["EllipticEnvelopeLearner"]
+
+
+class EllipticEnvelopeLearner(SklLearner):
+    __wraps__ = skl_covariance.EllipticEnvelope
+    name = 'elliptic envelope'
+
+    def __init__(self, store_precision=True, assume_centered=False,
+                 support_fraction=None, contamination=0.1,
+                 random_state=None, preprocessors=None):
+        super().__init__(preprocessors=preprocessors)
+        self.params = vars()
+        self.clf = None
+
+    def fit(self, X, Y=None, W=None):
+        self.clf = self.__wraps__(**self.params)
+        if W is not None:
+            return self.__returns__(self.clf.fit(X, W.reshape(-1)))
+        return self.__returns__(self.clf.fit(X))
+
+    def mahalanobis(self, observations):
+        """Return Mahalanobis distances of the training set observations.
```

Comment:
Is there a reason why we would not use the same as scikit-learn:
Computes the squared Mahalanobis distances of given observations.

(also below)